gamebot/Program.cs

```diff
@@ -224,20 +235,22 @@ public void Start()
 				});
 			_client.Ready += (s,e) =>
```

Comment:
here's the problem that made it not work: `Ready` never get activated (for me at least), so the whole save thing never works